### DISCUSSION TITLE
MedusaJS, Importing from Shopify

### DISCUSSION THREAD

**Discussion by nuckarronjones** [medusajs/medusa#7460] [Q&A]
Hello all, I had a question about migrating your shopify store to medusajs. From [this](https://medusajs.com/blog/shopify-migration/) help article, I am able to understand the migration instructions; however, does this require that the existing shopify store must continue to be active? Or, rather, when the migration takes place, the shopify storefront does not need to be active any longer. 

Thanks,

Nicholas

**Comment by abhishekrawat** [comment]
It must be active it seems, as you need to create an app. However i couldn't migrate the products even after creating it.
I see
>API Deprecation Notice: {
  >message: 'https://shopify.dev/api/admin-rest/latest/resources/product',
